src/pages/Login/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CommonForm from '../../../component/Form/CommonForm';
+import CommonInput from '../../../component/Input/CommonInput';
+import { LOGIN_INPUT_LIST } from './LOGIN_INPUT_LIST';
+
+import './LoginForm.scss';
+
+class LoginForm extends React.Component {
+  render() {
+    const loginInpuList = LOGIN_INPUT_LIST.map(elem => {
+      const { type, name, placeholder } = elem;
+      return <CommonInput type={type} name={name} placeholder={placeholder} />;
+    });
```

Comment:
```suggestion
    const loginInpuList = LOGIN_INPUT_LIST.map(({type, name, placeholder})=> 
       <CommonInput type={type} name={name} placeholder={placeholder} />;
    );
```
매개변수 자리에서 바로 구조분해 할수도 있습니다! 참고사항으로 전달드립니다.